### DISCUSSION TITLE
Latest info on WinAppSDK & WinUI: 1.2.3, 1.3, & February community call

### DISCUSSION THREAD

**Discussion by gabbybilka** [microsoft/microsoft-ui-xaml#8124] [Announcements]
Hello everyone! 👋

Welcome to the new year! As you may have noticed, this month we did not end up hosting a community call as initially announced in November. In lieu, here are a few updates about upcoming releases. 
- WinAppSDK 1.2.3: A servicing release fixing critical bugs for 1.2 planning to go live this week. 
  - *Update: https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.2.3
- WinAppSDK 1.3: As mentioned on previous community calls, 1.3 is centered around improving how you use WinUI & WinAppSDK by fixing quality issues, improving debuggability, and simplifying difficult to use APIs like Mica & Window/AppWindow. We are looking toward releasing the first prerelease in a few weeks and targeting April for the full stable release.

We look forward to answering your questions and sharing the latest updates with you at our next community call on February 15th at 9am PT (UTC-8): https://github.com/microsoft/microsoft-ui-xaml/discussions/8137

**Comment by JanRajnoha** [comment]
How planning is going? Could we somehow join to help? And if don't, is somewhere place, where we could find, what will be included in 1.3?

**Comment by bogdan-patraucean** [comment]
Please update [this docs](https://learn.microsoft.com/en-us/windows/apps/develop/title-bar?tabs=wasdk) also to mention the support for custom titlebar on Windows 10 (added in previous versions)

This is what it says now: "Title bar customization APIs are currently supported on Windows 11 only. "

**Comment by Mysterious-Dev** [comment]
There is any chance to have an improvement for definition of minimum size with windows app sdk (like we can do with UWP) ?

https://learn.microsoft.com/en-us/uwp/api/windows.ui.viewmanagement.applicationview.setpreferredminsize?view=winrt-22621

**Comment by omikhailov** [comment]
Hello! Can we post questions for the call here or there will be a special announcement for this event?

**Reply by gabbybilka** [reply]
Just posted a thread for you to leave questions for the February call! https://github.com/microsoft/microsoft-ui-xaml/discussions/8137
